lib/Distribution/Alp.pm

```diff
@@ -0,0 +1,16 @@
+# Summary: The class represents ALP distribution and provides access to
+# its features.
+
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Distribution::Alp;
+use strict;
+use warnings FATAL => 'all';
+use parent 'susedistribution';
+use YaST::Firewall::FirewallController;
+
+sub get_firewall {
```

Comment:
we don't need this method, neither a controller.

Reply:
Updated